client/src/pages/Login.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FunctionComponent } from "react";
+import { withTheme } from "@material-ui/core/styles";
+import Box from "@material-ui/core/Box";
+import styled from "styled-components";
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+// import Typography from '@material-ui/core/Typography';
+
+
+interface LoginProps {
+  className?: string;
+}
+
+/**
+ * Login Page.
+ * TODO:
+ * - form 형식 만들기
+ * - input data 와 passport middleware 연결
+ */
+const UnstyledLogin: FunctionComponent<LoginProps> = ({
+  className,
+}) => {
+  return (
+    <Box mt={20}>
+      <Grid container justify="center" alignItems="center" direction="column" spacing={5}>
+        <Grid item>
+          <h1> Login</h1>
+        </Grid>
+
+        <Grid item style={{ border: '0.2px solid gray' }}>
+          <Grid container direction="column" alignItems="center" justify="center">
+            <TextField variant="filled" color="secondary" type="email" label="Email:" fullWidth style={{ marginBottom: '2em', width: 500 }} />
```

Comment:
don't use inline styles when you can. You have bunch of examples in other files. 
Aslo `MUI` provides `<Box>` component that accepts those as props.

Reply:
should I use `<Box>` instead of `<Grid>`?
로그인 form을 화면 가운데에 위치시키려다 보니까 `<Grid>`를 사용했었습니다.
<img width="2558" alt="Screen Shot 2021-09-20 at 9 12 34 PM" src="https://user-images.githubusercontent.com/47313362/134097459-7881cf6e-1d72-4704-abf8-eedd3c7344d9.png">

Reply:
just use Box with flex styling

Reply:
네! I'll do my BEST!! 🙆‍♂️